roles/collect/tasks/main.yml

```diff
@@ -90,22 +84,11 @@
   changed_when: false
   no_log: true
 
-- name: Log in to OCP cluster
```

Comment:
Does OCP 4 no longer require logins? How does authentication happen?

Reply:
We don't login ,but instead use an auth token in the direct API call to the operator metering report API.